server/mediaserver.go

```diff
@@ -587,11 +592,18 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		seq = 0
 	}
+
+	duration, err := strconv.ParseFloat(r.Header.Get("Content-Duration"), 64)
```

Comment:
The idea of was to pass in an integer number of milliseconds, then convert to float as necessary at the last minute.  Something like:

```golang
duration, err := strconv.Atoi(r.Header.Get("Content-Duration"))
if err != nil {
  duration = 2000 // maybe do some reasonable default rather than zero?
  glog.Info("MIssing duration; filling in a default of 2000ms")
}
seg := &stream.HLSSegment {
   ....
   Duration: float64(duration)/1000.0,
}

Reply:
I'm fine with that. Why does it make a difference to convert at the last minute, rather than when first extracted from `Content-Duration`?

Reply:
If we ever need to use the value somewhere else, it's more helpful to maintain in integers rather than float.